src/components/LoadingSpinner.tsx

```diff
@@ -2,9 +2,9 @@ import React from "react";
 import { View, ActivityIndicator, StyleSheet } from "react-native";
 import { useTheme } from "react-native-paper";
 
-export const LoadingSpinner = ({size = 40  }) => {
+export const LoadingSpinner = ({size = 40, style  }) => {
   const theme = useTheme()
-  return (<View style={styles.container}>
+  return (<View style={{...styles.container, ...style}}>
```

Comment:
styles and style are very misleading names, please change style to something else, at least make this prop "sx" like a MUI convetion.

Reply:
sx it is